src/pages/RegistrationPage.js

```diff
@@ -194,8 +193,6 @@ class RegistrationPage extends React.Component {
         [FORM_ERROR]: generalMessage,
       }
     }
-    logDebug(this.props.context.store)
```

Comment:
Logging only happens in logDebug when NODE_ENV="development" so not sure we do need to remove this stuff, although I'm not sure this needs to be here.

Reply:
Logging like this should be ad-hoc. Its more of a mess to pop open the console and see these logs when your trying to debug something else on the page. I don't think we need this on master

Reply:
These were all front end logging, if they're needed we can add them locally!